app/src/interfaces/useProjectApi.interface.ts

```diff
@@ -91,6 +91,11 @@ export interface IGetUserProjectsListResponse {
  * @interface IGetProjectsListResponse
  */
 export interface IGetProjectsListResponse {
+  projectData: IProjectsListData;
+  projectSupplementaryData: { has_unpublished_content: boolean };
```

Comment:
`IProjectSupplementaryData` could potentially be its own interface

Reply:
Ditto for `ISurveySupplementaryData`

Reply:
done